src/sphinx_book_theme/__init__.py

```diff
@@ -164,6 +164,12 @@ def run(self):
         return nodes
 
 
+def update_general_config(app):
+    theme_dir = get_html_theme_path()
+    # Update templates for sidebar
+    app.env.config.templates_path.append(os.path.join(theme_dir, "components"))
```

Comment:
Can you add a comment explaining why this is needed given that we already add the components at the link below?

https://github.com/executablebooks/sphinx-book-theme/pull/566/files#diff-323e2d703544f60dac082d2b47d8b6a2d8e1594eb3b8f49d569f48f768379252R205

Otherwise it will confuse people why we do this twice and not just once...

Reply:
sure thing @choldgraf, I will add the explanation.